### PR TITLE
[BTAT-9201] - Update Capture Email View to include new GOVUK Assets

### PR DESCRIPTION
**Pull request by william-hatzar**

### CODE REVIEW

**Comment by william-hatzar** [comment]
AT's https://github.com/hmrc/change-vat-details-acceptance-tests/pull/154

File: app/views/email/CaptureEmailView.scala.html [hunk 1]
```diff
@@ -14,69 +14,90 @@
...
 
-@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
 @import config.AppConfig
 @import views.html.templates.inputs.Text
 @import views.html.templates.errors.ErrorSummary
 @import play.api.mvc.Call
 
-@this(mainTemplate: MainTemplateOriginal, form: FormWithCSRF, errorSummary: ErrorSummary, text: Text)
+@this(mainTemplate: MainTemplate, form: FormWithCSRF, govukErrorSummary: GovukErrorSummary, govukButton: GovukButton,
+govukBackLink: GovukBackLink, text: Text, govukInput : GovukInput, govukDetails : GovukDetails, formWithCSRF: FormWithCSRF)
```
Comment (lewishay) [inline app/views/email/CaptureEmailView.scala.html:23]: Can you remove the `Text` inject from here as this is the old text input component that will be deleted soon

File: app/views/email/CaptureEmailView.scala.html [hunk 2]
```diff
@@ -14,69 +14,90 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
 @import config.AppConfig
 @import views.html.templates.inputs.Text
```
Comment (lewishay) [inline app/views/email/CaptureEmailView.scala.html:18]: Can you remove the `Text`, `ErrorSummary` and `Call` imports

File: app/views/email/CaptureEmailView.scala.html [hunk 3]
```diff
@@ -14,69 +14,90 @@
...
-                        </a> @messages("captureEmail.removeEmail.description")
-                    </div>
-                </details>
-            </div>
-        }
+@if(!(currentEmail == "")) {
+    @govukDetails(Details(
+    summary = Text(messages("captureEmail.removeEmail.title")),
+    content = HtmlContent(
+    s"""<a target="_blank" href="${appConfig.contactHmrcUrl}">
```
Comment (lewishay) [inline app/views/email/CaptureEmailView.scala.html:89]: Can you add the attribute `rel="noreferrer noopener"` to this link

File: app/views/email/CaptureEmailView.scala.html [hunk 4]
```diff
@@ -14,69 +14,90 @@
...
+@if(currentEmail == "") {
+    <p class="govuk-body">@messages("captureEmail.onlyAddEmail")</p>
+}
 
-            <div class="form-group">
-                <button class="button" type="submit">@messages("common.continue")</button>
-            </div>
-        }
+@govukButton(Button(
+    content = Text(messages("common.continue")),
```
Comment (lewishay) [inline app/views/email/CaptureEmailView.scala.html:81]: Could you remove the comma from the end of this line as there's no param after it

**Review by lewishay — CHANGES_REQUESTED** [review]

**Pull request merged**